src/smallmap_gui.cpp

```diff
@@ -1751,6 +1751,15 @@ class NWidgetSmallmapDisplay : public NWidgetContainer {
 	{
 		for (NWidgetBase *child_wid = this->head; child_wid != nullptr; child_wid = child_wid->next) child_wid->Draw(w);
 	}
+
+	void FillDirtyWidgets(std::vector<NWidgetBase *> &dirty_widgets) override
+	{
+		if (this->is_dirty) {
+			dirty_widgets.push_back(this);
+		} else {
+			for (NWidgetBase *child_wid = this->head; child_wid != nullptr; child_wid = child_wid->next) child_wid->FillDirtyWidgets(dirty_widgets);
+		}
+	}
```

Comment:
I cannot seem to find where this stuff is used. Are there commits missing?

Reply:
gfx.cpp:1563 in this commit

Reply:
Ah, so it is. I wonder why KDevelop couldn't find it when I went looking for it 🤔. Perhaps I just need to find an IDE with fewer bugs in it?